update/qubes-dom0.sls

```diff
@@ -11,5 +11,4 @@
     - repl: qubesosfasa4zl44o4tws22di6kepyzfeqv3tg4e3ztknltfxqrymdad
 
 update:
-  pkg.uptodate:
-    - refresh: True
+  pkg.uptodate: []
```

Comment:
Sadly, this also isn't enough. It seems [pkg.uptodate state supports only refresh `True/False`](https://github.com/saltstack/salt/blob/master/salt/states/pkg.py#L3263), specifically not `None` which would make salt using rtag to decide. If it's set to False (the default), refreshing metadata is skipped, regardless of the rtag.
Maybe it's time for explicit metadata refresh in a separate state here?

Reply:
Darn :disappointed:.  Explicitly refreshing the metadata is probably the way to go here.